src/Contao/View/Contao2BackendView/ActionHandler/SelectHandler.php

```diff
@@ -390,7 +390,7 @@ private function handleCopyAllAction(EnvironmentInterface $environment, Action $
         foreach ($this->getModelIds($environment, $action, $this->getSubmitAction($environment)) as $modelId) {
```

Comment:
Unsure about this one, we might want to check again what `$action` is here. I suspect it could be `copy` or `deepcopy` already.
We now always enforce `deepcopy` here, while we before enforced `copy` - both is not as elegant.